sirepo/job_driver/__init__.py

```diff
@@ -310,6 +311,7 @@ def _receive_alive(self, msg):
 
         Save the websocket and register self with the websocket
         """
+        self._start_idle_timeout()
```

Comment:
I would put the start_idle_timeout inside the agent starting done.

Reply:
https://github.com/radiasoft/sirepo/pull/2804/commits/697d75b03e5b4d2b980c6307ba4ed4910673df0d